### PR TITLE
Fix compatibility matrix

### PR DESCRIPTION
**Pull request by nvazquez**
Compatibility matrix changes for release notes

### CODE REVIEW

**Comment by nvazquez** [comment]
@blueorangutan docbuild

File: source/releasenotes/compat.rst [hunk 1]
```diff
@@ -44,16 +44,17 @@ CloudStack supports three hypervisor families, XenServer with XAPI, KVM,
...
 -  Ubuntu 18.04 LTS, 20.04 LTS with KVM
--  CentOS 7, 8 with KVM (note: CentOS 8 will EOL in Dec 2021)
+-  CentOS 7 with KVM
 -  Rocky Linux 8 with KVM
 -  Red Hat Enterprise Linux 7, 8 with KVM
--  Rocky Linux 8 with KVM
 -  openSUSE Leap 15
 -  SUSE Linux Enterprise Server 15
--  XenServer versions 7.1, 7.2, 7.4, 7.5, 8.0 (8.1 and 8.2 expected to work, but not tested) with latest hotfixes, XCP-ng 7.4, 7.6, 8.0, 8.1, 8.2
+-  XenServer versions 7.1, 7.2, 7.4, 7.5, 8.0 (8.1 and 8.2 expected to work, but not tested. For 8.2 please check the note below) with latest hotfixes, XCP-ng 7.4, 7.6, 8.0, 8.1, 8.2.0
```
Comment (weizhouapache) [inline source/releasenotes/compat.rst:52]: @nvazquez 
should `.0` be added to all version numbers ?

```
mysql> select hypervisor_version from hypervisor_capabilities where hypervisor_type ='XenServer';
+--------------------+
| hypervisor_version |
+--------------------+
| 5.6                |
| 5.6 FP1            |
| 5.6 SP2            |
| 6.0                |
| 6.0.2              |
| 6.1.0              |
| 6.2.0              |
| 6.5.0              |
| 7.0.0              |
| 7.1.0              |
| 7.1.1              |
| 7.1.2              |
| 7.2.0              |
| 7.3.0              |
| 7.4.0              |
| 7.5.0              |
| 7.6.0              |
| 8.0.0              |
| 8.1.0              |
| 8.2.0              |
| default            |
| XCP 1.0            |
| XCP-ng 7.4.0       |
+--------------------+
23 rows in set (0.00 sec)
```

Comment (nvazquez) [reply source/releasenotes/compat.rst]: Thanks @weizhouapache done

**Comment by nvazquez** [comment]
@blueorangutan docbuild

**Comment by nvazquez** [comment]
@blueorangutan docbuild

**Review by weizhouapache — APPROVED** [review]
lgtm

**Comment by andrijapanicsb** [comment]
@nvazquez can we maybe remove "in this release" from the line which describes deprecated list of OS for hypervisor and mgmt servers:

_The following hypervisors are no longer be supported in this release due to vendor EOL_

i.e. there is already centos6, which was removed (support for it) in previous release, so perhaps reword to something like "The following hypervisors are not be supported due to vendor EOL" - or something similar - just to state that the list of OS-es is NOTf from this specific release - this is cumulative list of unsupported OS-es (do changes for both hypervisors AND for the mgmt server OS list).

Otherwise, LGTM.

**Pull request merged**